documentation/src/main/asciidoc/topics/con_performance_metric_considerations.adoc

```diff
@@ -0,0 +1,25 @@
+[id='performance-metric-considerations_{context}']
+= Performance metric considerations
+{brandname} includes so many configurable combinations that determining a single formula for performance metrics that covers all use cases is not possible.  
+
+The purpose of the _{brandname} performance considerations and tuning guidelines_ document is to provide details about use cases and architectures that can help you determine requirements for your {brandname} deployment.
```

Comment:
Hi! This is the community title but in the product it's different. This will require a follow up PR. Either you can reword it so you don't mention the title name or you can create an attribute for the title that will have different value in the community and in the product. I can help